openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/SearchIndexApplication.spec.ts

```diff
@@ -124,11 +116,7 @@ test('Search Index Application', async ({ page }) => {
     await page.click('[data-testid="submit-btn"]');
     await responseAfterSubmit;
 
-    await expect(page.getByRole('alert').first()).toHaveText(
-      /Configuration saved successfully/
-    );
-
-    await page.getByLabel('close').first().click();
+    await toastNotification(page, 'Configuration saved successfully');
```

Comment:
this one also regex

Reply:
same https://github.com/open-metadata/OpenMetadata/pull/16968#discussion_r1671672489